site/_docs/history.md

```diff
@@ -108,13 +108,13 @@ Runkang He,
 Tanner Clary,
 Thomas Rebele,
 Tim Nieradzik,
+Wang Zhao,
 Wegdan Ghazi,
 Wenrui Meng,
 Xiaogang Zhou,
 ZhangJian He,
 Zhengqiang Duan,
-Zoltan Haindrich,
```

Comment:
Sorry, Is `history.md` suitable for modification?

Reply:
IMO, it's ok to do it, there is no restrictions on the immutability of release notes.